LayoutTests/fast/scrolling/programmatic-scroll-merge-delta-and-position-expected.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Only the green part of the div should be visible</title>
+    <style>
+        body {
+            height: 2000px;
+            background-image: repeating-linear-gradient(white, silver 200px);
+            margin: 0px;
+        }
+        
+        .box {
+            width: 200px;
+            height: 200px;
+            background-image: linear-gradient(0deg, green 50%, red 50%);
+        }
+    </style>
+    <script src="../../resources/ui-helper.js"></script>
+    <script>
+        if (window.testRunner)
+            testRunner.waitUntilDone();
+
+        async function doTest()
```

Comment:
I don't think this has to be `async` in any of the tests. There's no `await` anywhere. But that makes me wonder if they behave correctly with UI-side compositing.

Reply:
Now using `await UIHelper.ensurePresentationUpdate()`.